packages/forklift-console-plugin/locales/en/plugin__forklift-console-plugin.json

```
@@ -391,6 +390,7 @@
   "URL must start with https:// or http:// and contain valid hostname and path": "URL must start with https:// or http:// and contain valid hostname and path",
   "URL of the provider": "URL of the provider",
   "URL of the provider, leave empty to use this providers URL": "URL of the provider, leave empty to use this providers URL",
+  "URL of the Red Hat Virtualization (RHV) engine API endpoint. Ensure it includes the \"/ovirt-engine/api\" path, for example: https://rhv-host-example.com/ovirt-engine/api": "URL of the Red Hat Virtualization (RHV) engine API endpoint. Ensure it includes the \"/ovirt-engine/api\" path, for example: https://rhv-host-example.com/ovirt-engine/api",
```

`engine` is an upstream term, if we refer to RHV here then it should be RHVM
```suggestion
  "URL of the Red Hat Virtualization Manager  (RHVM) API endpoint. Ensure it includes the \"/ovirt-engine/api\" path, for example: https://rhv-host-example.com/ovirt-engine/api": "URL of the Red Hat Virtualization Manager (RHVM) API endpoint. Ensure it includes the \"/ovirt-engine/api\" path, for example: https://rhv-host-example.com/ovirt-engine/api",
```

---

"Ensure the URL..." instead of "Ensure it" 

Either "...path. For example:..." or "...path, for example,..."